src/containers/NoAccessPage/NoAccessPage.js

```diff
@@ -58,6 +60,12 @@ export const NoAccessPageComponent = props => {
         heading: 'NoAccessPage.initiateTransactions.heading',
         content: 'NoAccessPage.initiateTransactions.content',
       }
+    : isViewingRightsPage
+    ? {
+        schemaTitle: 'NoAccessPage.viewListings.schemaTitle',
+        heading: 'NoAccessPage.viewListings.heading',
+        content: 'NoAccessPage.viewListings.content',
+      }
```

Comment:
There's probably a mismatch between this and Aleksi's PR.
Perhaps you should rebase your branch on top of it? Although, Aleksi might merge his branch soonish anyway.

Reply:
Rebased on the most recent main and updated to include ctaData